ScratchBlocks.php

```diff
@@ -61,17 +52,18 @@ function sbSetup () {
 
 $wgResourceModules['ext.scratchBlocks'] = array(
     'scripts' => array(
-        'scratchblocks2/build/scratchblocks2.js',
-        'run_scratchblocks2.js',
+        'ScratchBlocks/src/scratchblocks.js',
+		'scratchblocks/src/translations.js',
+        'run_scratchblocks.js',
     ),
 
-    'styles' => 'scratchblocks2/build/scratchblocks2.css',
+    'styles' => 'ScratchBlocks/src/defs.css',
```

Comment:
We don't want this here. These definitions are included in the JS (and then in the generated SVGs). :)